secure/key-manager/src/tests.rs

```diff
@@ -311,16 +506,26 @@ fn test_manual_consensus_rotation() {
 #[test]
 // This verifies the key manager is properly setup and that a basic rotation can occur
 fn test_key_manager_init_and_basic_rotation() {
+    // Test the mock libra interface implementation
+    let (config, _genesis_key) = config_builder::test_config();
+    let node = setup_node_using_test_mocks(&config);
+    verify_init_and_basic_rotation(node);
+
+    // Test the json libra interface implementation
     let (config, _genesis_key) = config_builder::test_config();
-    let mut node = Node::setup(&config);
-    node.key_manager.compare_storage_to_config().unwrap();
-    node.key_manager.compare_info_to_config().unwrap();
+    let (node, _runtime) = setup_node_using_json_rpc(&config);
+    verify_init_and_basic_rotation(node);
+}
+
+fn verify_init_and_basic_rotation<T: LibraInterfaceTestHarness>(mut node: Node<T>) {
+    // Verify correct initialization (on-chain and in storage)
+    assert!(node.key_manager.compare_storage_to_config().is_ok());
+    assert!(node.key_manager.compare_info_to_config().is_ok());
     assert_eq!(node.time.now(), node.key_manager.last_rotation().unwrap());
-    // No reconfiguration yet
     assert_eq!(0, node.key_manager.last_reconfiguration().unwrap());
-    // No executions yet
```

Comment:
?

Reply:
I think it was self explanatory, hence why I removed it, but I've added it back in :P